packages/resource-deployment/runtime-config/runtime-config.ci.json

```diff
@@ -0,0 +1,16 @@
+{
+    "logInConsole": true,
```

Comment:
can u also validate that this config file is loaded by convict.
You also need to update the ServiceConfiguration.profilePath property value accordingly.

Reply:
I'm wondering, should we have separate PRs, for uploading configuration and consuming the configuration? My PR is only doing the uploading part, so convict won't load it right?